src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+/**
+ * Represents whether a tag has been added or remved for a person in the current session
+ */
+public class Tagging {
+    
+    private final Person person;
+    private final Tag tag;
+    private boolean isTagRemoved;
```

Comment:
Using boolean is fine now since this LO requires only two states. What if later you need to capture more than two states?